src/definition-finder.ts

```diff
@@ -0,0 +1,49 @@
+'use babel';
+
+import MouseMoveListener from './mouse-move-listener';
+import {SourceRange} from 'polymer-analyzer/lib/model/model'
+
+class DefinitionFinder extends MouseMoveListener {
```

Comment:
High level comment, unsure if it's actionable, but doing this as a mouse move listener seems like the wrong level of abstraction. Does Atom have a concept of Go To Declaration/Definition? I see an entry in the right-click context menu for it. It seems like we should be able to tell Atom "hey I might know how to find the definitions for things" and then when the user (through their preferred mechanism, maybe a hotkey, maybe a mouse action) asks for the definition of something then Atom calls us and we return a location.